HousingManagementSystemApi/UseCases/RetrieveAddressesUseCase.cs

```diff
@@ -32,16 +35,23 @@ public async Task<IEnumerable<PropertyAddress>> Execute(string postcode)
 
             if (postcode == "")
                 return new List<PropertyAddress>();
-            var result = await addressesGateway.SearchByPostcode(postcode);
+            var result = await this.addressesGateway.SearchByPostcode(postcode);
```

Comment:
Nitpick: Please remove `this` qualifier
```suggestion
            var result = await addressesGateway.SearchByPostcode(postcode);
```